docs/spicedb/updating.md

```diff
@@ -1,57 +1,70 @@
-Join the [SpiceDB Discord] to be notified of new releases of SpiceDB.
+- SpiceDB Serverless upgrades seamlessly to track the latest release (and sometimes newer)
+- SpiceDB Dedicated users can select the desired version of SpiceDB
 
-[SpiceDB Discord]: https://authzed.com/discord
+No matter which service you select, zero-downtime migrations are always performed.
 
-### Checking for updates
+[managed service]: https://authzed.com/pricing
 
-Releases of SpiceDB can be found on the [SpiceDB Releases] page on GitHub.
+### Operator
 
-[SpiceDB Releases]: https://github.com/authzed/spicedb/releases
+If you are operating SpiceDB yourself, the recommended update workflow is to use the [SpiceDB Operator].
+Please see the [operator-specific update docs] for various update options.
 
-## Applying an update
+[SpiceDB Operator]: /spicedb/operator
+[operator-specific update docs]: /spicedb/operator#updating-managed-spicedbclusters
 
-Applying an update for SpiceDB requires three steps:
+### Sequential Updates
 
-### 1. Download new release of SpiceDB
+Thus, we highly recommend updating sequentially through SpiceDB minor versions (e.g. 1.0.0 -> 1.1.0) to avoid headaches.
+Jumping many versions at once might cause you to miss instructions for a particular release that could lead to downtime.
+The [SpiceDB Operator](#operator) automates this process.
 
-Download the new release from the [SpiceDB Releases] page or from [DockerHub]
+### Rolling Deployments
 
-[DockerHub]: https://hub.docker.com/r/authzed/spicedb
+We highly recommend a deployment orchestrator to help coordinate rolling out new instances of SpiceDB without dropping traffic.
+SpiceDB has been developed with an eye towards running on Kubernetes, but other platforms should be well supported.
 
-### 2. Migrate the datastore
+## Migrations
 
-Run `spicedb migrate head` to migrate the existing cluster's datastore forward:
+### What are migrations?
 
-```sh
-spicedb migrate head
-```
+For all software that maintains data, there can be updates to code that rely on also updating the data, too.
+The process of updating data to use new versions of software is called _migrating_.
+
+SpiceDB users will see migrations crop up in two places: when they update versions of SpiceDB and when they make backwards incompatible to their own schemas.
+This document's focus is on the former.
 
-:::note
-**Unless otherwise documented**, this is a backward compatible change
-:::
+### SpiceDB migration tooling
 
-### 3. Deploy the new release of SpiceDB
+SpiceDB ships with a migration command: `spicedb migrate`.
+This command powers all of Authzed products' zero down-time migrations, so it's guaranteed to be battle-tested and supportable.
+And while you are free to explore other tools to help you migrate, we cannot recommend them.
 
-Deploy the new release of SpiceDB using your deployment system.
+If you do not care about causing downtime or you are bringing up a new cluster, you can always run the following command to migrate:
```

Comment:
I'd clarify without checking release notes; often this will work without downtime